corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -226,15 +249,18 @@
 
   </Target>
 
-
   <!-- Computes the managed assemblies that are input to the
        linker. Includes managed assemblies from
        ResolvedAssembliesToPublish, and IntermediateAssembly. -->
   <Target Name="_ComputeManagedAssembliesToLink"
-          DependsOnTargets="_ComputeManagedResolvedAssembliesToPublish">
+          DependsOnTargets="_ComputeManagedResolvedAssembliesToPublish;_ComputePlatformLibraries">
     <ItemGroup>
       <_ManagedAssembliesToLink Include="@(IntermediateAssembly)" />
       <_ManagedAssembliesToLink Include="@(_ManagedResolvedAssembliesToPublish)" />
+      <_PlatformAssembliesToLink Include="@(PlatformLibraries->'%(Filename)')" />
+      <_PlatformAssembliesToLink Include="System.Private.CoreLib" />
+      <_ApplicationAssembliesToLink Include="@(_ManagedAssembliesToLink->'%(Filename)')" />
+      <_ApplicationAssembliesToLink Remove="@(_PlatformAssembliesToLink)" />
```

Comment:
Could you compute these itemgroups in the target `_SetAssemblyActions` (and make `_ComputePlatformLibraries` a dependency of `_SetAssemblyActions` instead), since they're only needed by that logic? There's at least one case in which `_ComputeManagedAssembliesToLink` is called, but `_SetAssemblyActions` isn't (during an incremental build when the linker inputs are unchanged, `_ManagedAssembliesToLink` is used for the comparison report).

Reply:
Done.